src/presetsController.ts

```diff
@@ -590,7 +590,7 @@ export class PresetsController {
         interface PresetItem extends vscode.QuickPickItem {
             preset: string;
         }
-        const presetsPool: preset.Preset[] = showHiddenPresets ? candidates : candidates.filter(_preset => !_preset.hidden && preset.evaluatePresetCondition(_preset, allPresets));
+        const presetsPool: preset.Preset[] = showHiddenPresets ? candidates : candidates.filter(_preset => !_preset.hidden && preset.evaluatePresetCondition(_preset, allPresets) !== undefined);
```

Comment:
I don't believe this is the correct fix.  If the condition is false, we shouldn't return true.